docs/troubleshooting-guide/models.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Troubleshooting models
+---
+
+# Troubleshooting models
+
+What kind of problem are you having with your [model][model-docs]?
+
+- [Cannot create a model](#cannot-create-a-model).
+- [Cannot edit or save changes to a model](#cannot-edit-or-save-changes-to-a-model).
+- [Model performance is poor](#model-performance-is-poor).
+- [Model doesn't work with data sandboxing][troubleshooting-sandboxing].
+
+## Cannot create a model
+
+If you don't see [the model button][model-button-image] (three squares):
```

Comment:
I put a link to the Learn image here (instead of adding it inline)... any objections? 👀

Reply:
Risky, but okay for now. I'd like to update the model page to include an image pointing out where all this stuff is, though it looks like the UI will change to a dropdown menu for 44. No need to worry about that here, just noting it.